jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -238,8 +238,16 @@ class JWTTools(
     ): JwtPayload {
         val (header, payload, signatureBytes) = decode(token)
 
-        if (payload.iat != null && payload.iat > (timeProvider.nowMs() / 1000 + TIME_SKEW)) {
-            throw InvalidJWTException("Jwt not valid yet (issued in the future) iat: ${payload.iat}")
+        val nowSkewed = (timeProvider.nowMs() / 1000 + TIME_SKEW)
+
+        if (payload.nbf != null) {
```

Comment:
Not quite. Using your suggestion, if `payload.nbf` is in the past it would go ahead and compare using `iat` which is not the desired effect.
The desire is that nbf takes over completely if it exists.